src/components/Song/Upvote.js

```diff
@@ -1,26 +1,23 @@
 import { contains } from 'ramda'
 import React from 'react'
-import Arrow from '../../img/arrow-up.svg'
 import { readVotesFromStorage } from 'util/votes'
 
-const isUpvoted = songId => {
-  let persistedVotes = readVotesFromStorage()
-  return contains(songId, persistedVotes)
-}
-
 const Upvote = ({changeVote, songId}) => {
-  if (isUpvoted(songId)) return <div>VOTED!</div>
+  const isUpvoted = contains(songId, readVotesFromStorage())
+  const styles = {
+    arrowStyles: {
+      width: '50px',
+      fill: isUpvoted ? 'orangered' : ''
+    }
+  }
   return (
-    <div>
-      <img alt={'vote-up'} src={Arrow} onClick={() => changeVote(1)} style={styles.arrowStyles} />
+    <div onClick={() => changeVote(isUpvoted ? -1 : 1)}
+         style={styles.arrowStyles}>
```

Comment:
curious whether we could move these props to the `<svg>` and get rid of the `<div>` entirely.

Reply:
Yeah.. for some reason it threw an error when I had the old svg format and I had moved these into the div to try to fix. Works fine on the svg now though